packages/wonder-blocks-banner/src/components/banner.tsx

```diff
@@ -445,7 +477,15 @@ const styles = StyleSheet.create({
         justifyContent: "center",
     },
     link: {
-        fontSize: font.body.size.small,
+        fontFamily: bannerTokens.link.font.family,
+        fontSize: bannerTokens.link.font.size,
+        fontWeight: bannerTokens.link.font.weight,
+        lineHeight: bannerTokens.link.font.lineHeight,
+        textDecoration: bannerTokens.link.font.decoration,
+        textUnderlineOffset: bannerTokens.link.font.underlineOffset,
+    },
+    button: {
+        marginInline: bannerTokens.button.layout.marginInline,
```

Comment:
We set a negative inline margin in TB to account for the padding in the tertiary Button. This makes the text in the Button line up with the banner text when there are multiple lines of text